examples/quarkus-jib-extension/src/main/java/org/acme/GreetingResource.java

```diff
@@ -0,0 +1,16 @@
+package org.acme;
```

Comment:
The package name will need to be modified to `example`. Here is an example: https://github.com/GoogleContainerTools/jib/blob/master/examples/helloworld/src/main/java/example/HelloWorld.java